lib/utils/new-module.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const EMBER_NAMESPACES = ['inject.controller', 'inject.service'];
+
+function buildMessage(obj) {
```

Comment:
This function is too long and complicated and should have simple helper functions extracted from it.

Reply:
This is existing (and longer) code that I've extracted to an util to be reused between two rules. With that explanation, do you think I should implement your suggestion?